testutils/utils.go

```diff
@@ -59,6 +61,22 @@ func WipeEtcd() {
 		panic(err)
 	}
 	_ = be.Clean()
+
+	// Set the ready flag so calls to the CNI plugin can proceed
+	calicoClient, _ := client.NewFromEnv()
+	ctx := context.Background()
+	newClusterInfo := api.NewClusterInformation()
+	newClusterInfo.Name = "default"
+	newClusterInfo.Spec.CalicoVersion = "v0.0.0"
+	newClusterInfo.Spec.ClusterType = "test"
+	newClusterInfo.Spec.ClusterGUID = fmt.Sprintf("%s", hex.EncodeToString(uuid.NewV4().Bytes()))
```

Comment:
I don't think we need to populate any other `Spec` fields since they're not used in CNI. Especially this one, it'll save us an extra import